html_css/Exercise_1/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Exercise_1</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+  </head>
+  <body>
+    <div class="white_box">
```

Comment:
Make sure proper errors are raised when we try to submit this form when keeping fields blank